projects/driver_controls/inc/steering_wheel.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
This module should be called steering angle or similar. You're not interfacing with the steering wheel.